src/wp-admin/widgets-form.php

```diff
@@ -298,25 +298,25 @@
 	<div class="widget-position">
 	<table class="widefat"><thead><tr><th><?php _e( 'Sidebar' ); ?></th><th><?php _e( 'Position' ); ?></th></tr></thead><tbody>
 	<?php
-	foreach ( $wp_registered_sidebars as $sbname => $sbvalue ) {
-		echo "\t\t<tr><td><label><input type='radio' name='sidebar' value='" . esc_attr( $sbname ) . "'" . checked( $sbname, $sidebar, false ) . " /> $sbvalue[name]</label></td><td>";
-		if ( 'wp_inactive_widgets' === $sbname || 'orphaned_widgets' === substr( $sbname, 0, 16 ) ) {
+	foreach ( $wp_registered_sidebars as $sidebar_name => $sidebar_data ) {
```

Comment:
Similar as above. Check same instance in file.
```suggestion
	foreach ( $wp_registered_sidebars as $sidebar => $registered_sidebar ) {
```

Reply:
I'm not sure about this one as `$sidebar_name` is more descriptive of what the variable contains.

`$sidebar_data` could be renamed to `$registered_sidebar`, but I'm not sure that this is clearer than `$sidebar_data`.